src/client/src/utils/firebase/index.js

```diff
@@ -0,0 +1,14 @@
+import { initializeApp } from "firebase/app";
```

Comment:
Should we upload this as well? Or are there secrets in here as well

Reply:
This is the import from the [firebase API](https://firebase.google.com/docs/reference/js/v8/firebase#initializeapp)